docs/reduction-in-locks.md

```diff
@@ -0,0 +1,98 @@
+    | `.crpt`      | For files that cannot be fully copied due to encryption changes. These will be recopied, and a `.crpt` file will be created for the existing file. |
+    | `.new.meta`  | Similar to `.new`, but for incremental backups.                                                                  |
+    | `.new.delta` | Similar to `.new`, but for incremental backups. Create `t1.ibd.meta` and `t1.ibd.delta` instead of `t1.ibd`.      |
+
+4. Gather a sync point from all engines (InnoDB LSN, binlog, GTID, etc.) by querying the `log_status`.
+5. Stop the redo flow thread once it copies at least up to sync point at step 4.
+6. Release the backup lock (LTFB/LIFB).
+
+### Phase 3. Processing the new metadata files during the `--prepare` phase before crash recovery starts.
+
+1. The `.crpt` files are removed matching the name after stripping the extension. This step is crucial before the IBD scan because these files are incomplete (they could even be zero size).
+2. Perform a scan to create a mapping of `space_id` to file names.
+3. The `space_id.del` deletes the file matching the `space_id`. In case of incremental backups, also deletes the corresponding `.new.meta` and `.new.delta` files.
+4. The `space_id.ren` renames the file matching the `space_id` to the name specified in the file.
+5. The `.new` extension replaces the file that matches the name without the `.new` extension.
+6. The `.new.meta/.delta` replace the meta and delta files that match the name without the `.new` in the name.
+
+After Phase 3 is completed, the regular crash recovery starts.
+
+### Limitations
+
+1. The `ALTER INSTANCE ROTATE MASTER KEY` command should not be executed during [Phase 1](#phase-1-operations-performed-without-the-lock) if there are encrypted tables, as it will cause the backup to fail.
+
+2. The number of open file handles in your Operation System (OS) should be configured to match the number of files in the server data directory.
+
+3. When using the `--lock-ddl=REDUCED` option, backups may be larger because new `DDL` operations are executed concurrently on the server, and the files they generate are included in the backup. Additionally, certain DDL operations, such as `ADD INDEX` or encryption changes to existing data files, will cause these data files to be recopied, further increasing the backup size.
+
+### Benefits
+
+The `--lock-ddl=REDUCED` option key features are as follows:
+
+* Reduced locking: Rather than holding the backup lock throughout the entire backup process, it is only acquired for a very short period.
+
+* DDL statements: The server remains accessible for `DDL` operations during the backup process.
+
+* Consistency: Consistency is maintained by using the redo log to track file operations. During the backup process, new metadata files are created to ensure consistency. In the `--prepare` phase, these metadata files are processed first to bring the database to a consistent state.
```

Comment:
Consistency: Consistency (?) reword this

Reply:
May we be can rewrite as simple paragraph. 
DDL operations are allowed on the server while the backup is in progress with --lock-ddl=reduced...

Reply:
updated